clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperation.java

```diff
@@ -61,19 +64,27 @@ private static Task getTask() {
   public DeploymentResult operate(List priorOutputs) {
     updateTaskStatus("Initializing Create Amazon ECS Server Group Operation...");
 
+    String region = getRegion();
+    String credentialAccount = description.getCredentialAccount();
```

Comment:
Declaring the variables at this level is unnecessarily polluting this context, since they are not going to be used locally, and bloating the signatures of the methods called here with information that is readily available from the object's internal variables. 

Now this method has the added responsibility to initializing variables that are going to be used somewhere else on top of controlling the logic for the operation.